src/index.js

```diff
@@ -4,36 +4,33 @@ const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
 const storage = require('./statuscakedata');
 
-const regex = /\s?([-])\s?/;
+const { tagsFromTestTitleRegexp } = config.statuscake;
```

Comment:
```suggestion
const { tagsFromTestTitleRegexp, nameOfLoadTimeGauge, tags, nameOfFileSizeGauge, nameOfRequestGauge } = config.statuscake;
```
You can get all the varaibles from statuscake here

Reply:
Thanks ✌️